src/components/Node/Node.scss

```diff
@@ -2,9 +2,9 @@
   box-sizing: border-box;
   position: absolute;
   transition: all ease-in-out 0.2s;
-  background-color: #eee;
+  background-color: rgb(250,238,222);
```

Comment:
not sure about this, at the minute, still taking the stance of using colour where meaning needs to be conveyed.